### ISSUE TITLE
Enhance wins JavaScript to assign alt text to GitHub icon

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#4088]
### Overview

To comply with Web Content Accessibility Guidelines (WCAG), we need to ensure that image alt text on the Wins page is meaningful and appropriate for each icon. This issue addresses the need for alt text on the GitHub icon in the Wins cards, which are generated using JavaScript.

### Action Items

- [x] Understand the role of these components of the wins page:
  - [x] The main Wins web page: [wins.html](https://github.com/hackforla/website/blob/gh-pages/pages/wins/wins.html), which uses Liquid to include the template for each win card: [wins-card-template.html](https://github.com/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html)
  - [x] The JavaScript code [wins js](https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js) running in the Wins web page.
- [x]  In [wins-card-template.html](https://github.com/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html), find the img element with class 'github-icon', and add an empty alt attribute (alt='').
- [x] In [wins.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js), search for the text '.github-icon' in order to locate the line of code which sets the "src" attribute for the GitHub icon. Add code to set the "alt" attribute of the icon equal to "GitHub profile for name" where name is the name of the contributor. For example, the alt text for the image representing the link to Mary Smith's GitHub page should be "GitHub profile for Mary Smith".
- [x] A change will also be required in wins.js in the function updateOverlay to set the alt text in the Overlay (when the user clicks "See More" in wide-screen view).  
- [x] View the wins page to test the code at mobile, tablet and full-screen widths. Also also follow the "See More" page to check the alt tag on the modal overlay.

### Resources/Instructions

[Wins webpage](https://hackforla.org/wins/)
[wins.html](https://github.com/hackforla/website/blob/gh-pages/pages/wins/wins.html)
[wins js](https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js)
[wins-card-template.html](https://github.com/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html)
This issue was created as a follow-up to https://github.com/hackforla/website/issues/2569. See that issue for accessibility resources.

**Comment by ExperimentsInHonesty** [comment]
- @roslynwythe I unassigned you from this issue since it's in the icebox.  Is there are reason why it has the dependency https://github.com/hackforla/website/issues/4010?  It seems like they could be worked on at the same time?  Am I missing something?

**Comment by roslynwythe** [comment]
> * @roslynwythe I unassigned you from this issue since it's in the icebox.  Is there are reason why it has the dependency [Enhance wins JavaScript to assign alt text to LinkedIn icon #4010](https://github.com/hackforla/website/issues/4010)?  It seems like they could be worked on at the same time?  Am I missing something?

Yes @ExperimentsInHonesty they could be worked on at the same time, but to reduce the number of merge conflicts I added the dependencies.  I asked about this in a comment: https://github.com/hackforla/website/issues/2569#issuecomment-1436199120 and also asked the merge team about and they thought it was a good idea, but honestly after creating all the issues and dependencies I realize it probably wasn't worth the extra effort and confusion.  Should I remove all the dependencies and put them all back in New Issue Approval?

**Comment by jdingeman** [comment]
We'll remove the dependency and they can be worked on at the same time. Marking as ready for prioritization

**Comment by richardmundyiii** [comment]
Availability: 3/20/2023 7pm - 10pm
ETA: EOD 3/20/2023

**Comment by roslynwythe** [comment]
- @richardmundyiii I edited this issue to mention that to handle both mobile and large screen sizes, a change will be required in the function updateOverlay in wins.js

**Comment by richardmundyiii** [comment]
@roslynwythe No problem. I will be sure to update both _includes/wins-page/wins-overlay.html and the updateOverlay() function in wins.js
